compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -365,12 +365,15 @@ static SmallVector<int64_t> setOuterPermB(bool isMatmulTransposeB,
 
 static LogicalResult setRootConfigForPackPeel4LevelTilingPipeline(
     mlir::FunctionOpInterface entryPointFn, linalg::LinalgOp linalgOp,
-    AMDAIEDevice targetDevice, uint32_t numRows, uint32_t numCols) {
+    LowerToAIEPassPipeline useLowerToAIEPipeline, AMDAIEDevice targetDevice,
+    uint32_t numRows, uint32_t numCols) {
   // Scale the L1 K with a factor of 2 compared with the outer dimensions M and
   // N to increase the L1 memory usage.
+  bool isObjectFifo =
+      useLowerToAIEPipeline == LowerToAIEPassPipeline::ObjectFifo;
   auto maybePackPeelTiling =
-      ParameterSetting::create(linalgOp, /*isObjectFifo=*/true, targetDevice,
-                               numRows, numCols, /*kPackScaleL1=*/2);
+      ParameterSetting::create(linalgOp, isObjectFifo, targetDevice, numRows,
```

Comment:
Are you sure this is enough to get correct numerics? You have to modify the outer perm similar to this https://github.com/nod-ai/iree-amd-aie/blob/8ded4b1a84445cf5367c5db8f41435659c2dc941/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp#L582 inside `setRootConfigForPackPeel4LevelTilingPipeline`.

Reply:
If you like you can add a test in ci to make sure it's running.

Reply:
Oh cool, that would be great! Will add a matmul test to make sure.